pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -48,51 +48,44 @@ social:
+- entries << {name: "pre", text: "Pre-reader"}
+- entries << {name: "2-5", text: "Grades 2-5"}
+- entries << {name: "6-8", text: "Grades 6-8"}
+- entries << {name: "9+", text: "Grades 9+"}
 - filters << {name: "grade", text: "Grades", entries: entries}
 
 - entries = []
 - entries << {name: "beginner", text: "Beginner"}
 - entries << {name: "comfortable", text: "Comfortable"}
-- entries << {name: "experienced", text: "Experienced"}
-- filters << {name: "teacher_experience", text: "Educator's CS experience", entries: entries}
+- filters << {name: "teacher_experience", text: "Educator's experience", entries: entries}
 
 - entries = []
 - entries << {name: "beginner", text: "Beginner"}
 - entries << {name: "comfortable", text: "Comfortable"}
-- entries << {name: "experienced", text: "Experienced"}
-- filters << {name: "student_experience", text: "Student's CS experience", entries: entries}
+- filters << {name: "student_experience", text: "Student's experience", entries: entries}
 
 - entries = []
-- entries << {name: "online-tutorial", text: "Online tutorials"}
-- entries << {name: "lesson-plan", text: "Lesson plan"}
-- entries << {name: "offline-app", text: "Offline apps (for poor or no internet)"}
-- entries << {name: "unplugged", text: "Classroom activity with no technology"}
-- filters << {name: "activity_type", text: "Activity type", entries: entries}
+- entries << {name: "computers", text: "Computers"}
+- entries << {name: "android", text: "Android"}
+- entries << {name: "ios", text: "iPad/iPhone"}
+- entries << {name: "no-internet", text: "Poor or no internet"}
+- entries << {name: "no-computers", text: "No computers or devices"}
+- filters << {name: "platform", text: "Classroom technology", entries: entries}
 
 - entries = []
 - entries << {name: "science", text: "Science"}
 - entries << {name: "math", text: "Math"}
-- entries << {name: "history", text: "Social Studies, History"}
+- entries << {name: "history", text: "Social Studies"}
 - entries << {name: "la", text: "Language Arts"}
-- entries << {name: "language", text: "World language"}
 - entries << {name: "art", text: "Art, Design, Media"}
 - entries << {name: "other", text: "Other"}
 - entries << {name: "cs-only", text: "Computer Science only"}
 - filters << {name: "subject", text: "Topics", entries: entries}
 
 - entries = []
-- entries << {name: "all", text: "All devices"}
-- entries << {name: "browser", text: "Computers"}
-- entries << {name: "android-tablet", text: "Android Tablet"}
-- entries << {name: "ipad", text: "iPad"}
-- entries << {name: "android-phone", text: "Android phone"}
-- entries << {name: "iphone", text: "iPhones"}
-- entries << {name: "unplugged", text: "No computers or devices"}
-- filters << {name: "platform", text: "Classroom technology", entries: entries}
+- entries << {name: "online-tutorial", text: "Self-led tutorials"}
+- entries << {name: "lesson-plan", text: "Lesson plan"}
+- filters << {name: "activity_type", text: "Activity type", entries: entries}
```

Comment:
Does there need to be a robotics entry so that they never shows up in the all tutorials view?